modules/lightning_features/lightning_preview/lightning_preview.info.yml

```diff
@@ -8,3 +8,4 @@ experimental: true
 dependencies:
   - lightning_workflow
   - workspace
+components: {}
```

Comment:
SubProfileCommand::getLightningComponents() uses the presents of the components key to determine if this is a top-level components. Seems more logical than adding an additional "is_top_leve: true" value or something. But I'm open to suggestions.